terraform/alarms.tf

```
@@ -64,8 +64,8 @@ resource "aws_cloudwatch_metric_alarm" "sre_bot_high_cpu" {
   metric_name               = "CPUUtilization"
   namespace                 = "AWS/ECS"
   period                    = "60" # for 60 seconds
-  evaluation_periods        = "120"
-  statistic                 = "Average"
+  evaluation_periods        = "30"
+  statistic                 = "Maximum"
```

If we want this to be more sensitive, we could just do like so:
```suggestion
  evaluation_periods        = "1"
  statistic                 = "Maximum"
```

That would cause the alarm to trigger if the max CPU use goes over the threshold in a 1 minute period.  The way the alarm is setup with 30 eval periods, it would need to be consistently over the threshold each minute for 30 minutes.

---

Yes, because the AWS group check would trigger it otherwise and that is why I thought we should have more data points. So for the deployment failure, it was consistently over 80% for a long time. If we change it to 1, wouldn't we get alarm notifications every 2 hours or when the AWS group sync happens since at that time the CPU goes to 100%? I could be totally off though.

---

Ah right, that is a valid point about continuous alarm spikes if we always get 100% use during those sync jobs.

---

Lets try it with the 30 eval periods!

---

Thanks! We can always tweak it after as well.